relay/relay.go

```diff
@@ -19,6 +19,9 @@
 // THE SOFTWARE.
 
 // Package relay contains relaying interfaces for external use.
+//
+// These interfaces are currently unstable, and aren't covered by the API
```

Comment:
thanks for adding this. it's unrelated to this change, but if we don't already have this on Relay* fields in the channel options, can we also add it there.